data/Nigeria/Senate/sources/instructions.json

```diff
@@ -21,6 +21,21 @@
         "from": "group-wikidata",
         "source": "manual/group_wikidata.csv"
       }
-    }
+    },
+    {
+      "file": "morph/shineyoureye.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "struan/nigeria-shineyoureye",
+        "query": "SELECT *, NULL as position FROM data WHERE position = 'Senator'"
```

Comment:
I worry that just having `shineyoureye` may turn out to be confusing if / when we migrate to the static ShineYourEye site. Also, I'd like to include `slug` in the scheme, since it might well be useful to also include the primary key ID as well. So then you could have the two schemes:

* `source__shineyoureye__pombola_slug`
* `source__shineyoureye__pombola_id`

... perhaps?